rebasehelper/utils.py

```diff
@@ -730,11 +730,27 @@ def split_nevra(cls, string):
             return dict(name=name, epoch=epoch, version=version, release=release, arch=arch)
         raise RebaseHelperError('Unable to split string into NEVRA.')
 
+    @classmethod
+    def parse_spec(cls, path):
```

Comment:
I am not sure, if parse_spec is a correct name for this function, as it basically just reads the specfile and (for the meantime)gets rid of BuildArch tag lines. 
But considering the possible future spec parsing, we might leave it this way.

Reply:
Well, it calls `rpm.spec()` which does parse the spec file.

Reply:
Yes, then the name is appropriate.